pkgs/tools/misc/fortune/default.nix

```
@@ -1,26 +1,39 @@
-{ stdenv, fetchurl, recode }:
+{ stdenv, fetchurl, cmake, recode, perl }:
 
+let srcs = {
+      fortune = fetchurl {
+        url = https://github.com/shlomif/fortune-mod/archive/fortune-mod-2.6.2.tar.gz;
```

Make version a variable (in the let block)
```suggestion
        url = https://github.com/shlomif/fortune-mod/archive/fortune-mod-${version}.tar.gz;
```

---

The suggestion above still holds. If you exchange the version number with a variable the auto updater can make an attempt to bump the version (you may need to put quotes around the url for it to work).

---

I forgot this. Now it has been changed.